### ISSUE TITLE
Adicionar contributing

### ISSUE THREAD

**Issue by pabarros** [b2wads/nodejs-boilerplate#35]

**Comment by eduaugustus** [comment]
Hi, can i help??

**Comment by pabarros** [comment]
@eduaugustus Claro! Sinta-se livre para abrir uma PR com a sua contribuição :smile:

Atualmente o que temos de fluxo é forkar o projeto e abrir uma PR para a master com as alterações.

**Comment by eduaugustus** [comment]
@pabarros Vocês utilizam mais labels para caracterizar os PR, como label de bug e enhancement?

**Comment by pabarros** [comment]
@eduaugustus Não temos esse costume, porém seria algo legal para o projeto!

Talvez inciar utilizando as labels padrões do Github seja algo bacana.

**Comment by eduaugustus** [comment]
@pabarros show, vou fazer um esquema com algumas boas práticas que eu achar e que se adeque com o projeto!

**Comment by pabarros** [comment]
@eduaugustus ok! 

Obrigado pela disposição em contribuir :hugs:

**Comment by eduaugustus** [comment]
@pabarros estava pensando em fazer um **Contribution guidelines**, com algumas informações relevantes como começar a contribuir, além de fazer alguns templates para bug, enhancement e documentation. Penso em fazer algo simples, mas que não seja muito burocratico, mas que possa trazer alguns paadrões legais para o projeto, o que achas?

**Comment by pabarros** [comment]
@eduaugustus Hoje utilizamos um [template de PR](https://github.com/b2wads/nodejs-boilerplate/blob/master/.github/PULL_REQUEST_TEMPLATE.md) que acreditamos ser genérico o suficiente para compreender as alterações de bug e features. Talvez te ajude a definir os templates para contribuição.

Fique à vontade para sugerir o que achar relevante.
